app/views/wizards/claims/add_user_wizard/_check_your_answers_step.html.erb

```diff
@@ -0,0 +1,36 @@
+<% content_for :page_title, sanitize(t(".page_title", caption: contextual_text)) %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= form_for(current_step, url: current_step_path, method: :put) do |f| %>
+        <%= f.govuk_error_summary %>
+
+        <span class="govuk-caption-l"><%= contextual_text %></span>
+        <h1 class="govuk-heading-l"><%= t(".title") %></h1>
+
+        <%= govuk_summary_list(actions: false) do |summary_list| %>
+          <% summary_list.with_row do |row| %>
+            <% row.with_key(text: User.human_attribute_name("first_name")) %>
+            <% row.with_value(text: wizard.steps[:user].first_name) %>
+          <% end %>
+          <% summary_list.with_row do |row| %>
+            <% row.with_key(text: User.human_attribute_name("last_name")) %>
+            <% row.with_value(text: wizard.steps[:user].last_name) %>
+          <% end %>
+          <% summary_list.with_row do |row| %>
+            <% row.with_key(text: User.human_attribute_name("email")) %>
+            <% row.with_value(text: wizard.steps[:user].email) %>
+          <% end %>
+        <% end %>
```

Comment:
There are no 'change' links on this 'Check your answers' page 😕

I appreciate you're faithfully recreating the ['Check your answers' page](https://github.com/DFE-Digital/itt-mentor-services/blob/06fa2055d5da3eb6f26678c8c2c26a7fe9bb7fe9/app/views/claims/schools/users/check.html.erb#L23-L36) that school users saw before. But in this case I think that's a mistake, and would consider the lack of 'change' links for school users to be a bug. 🐛 Especially since [support users got 'change' links](https://github.com/DFE-Digital/itt-mentor-services/blob/06fa2055d5da3eb6f26678c8c2c26a7fe9bb7fe9/app/views/claims/support/schools/users/check.html.erb#L29-L40).

Let's take this opportunity to fix the bug now it's been uncovered. 🙏🏻